README.md

```diff
@@ -1,2 +1,70 @@
-# WagsTools
-Data acquisition tools for Wagnerds
+# WagsTails
+
+Data acquisition tools for Wagnerds.
+
+## Installation
+
+Install from PyPI:
+
+```shell
+python3 -m pip install wagstails
```

Comment:
? We can keep as is if you prefer it better

```suggestion
python3 -m pip install wags_tails
```

Reply:
now that you mentioned it, this would've solved the "wag stools" problem